api/src/main/java/org/openmrs/module/fhir2/api/FhirPersonService.java

```diff
@@ -25,4 +29,9 @@
 	Collection<Person> findPersonsByGender(String gender);
 	
 	Collection<Person> findPersonsByBirthDate(Date birthDate);
+	
+	Collection<Person> searchForPeople(StringOrListParam name, TokenOrListParam identifier, TokenOrListParam gender,
```

Comment:
Will it be a good idea to return a "Not Implemented" response (not sure how) to a query on `identifier` field? In case we plan to map the field in future.

Reply:
So if we just don’t provide an implementation of it at all (i.e., remove it from this method) the HAPI library will do that for us.